HousingSearchApi.Tests/Dockerfile

```diff
@@ -30,5 +34,7 @@ COPY . .
 RUN dotnet build -c Release -o out HousingSearchApi/HousingSearchApi.csproj
 RUN dotnet build -c debug -o out HousingSearchApi.Tests/HousingSearchApi.Tests.csproj
 
-CMD dotnet test --filter HousingSearchApi.Tests.V1
-RUN dotnet sonarscanner end /d:sonar.login="${SONAR_TOKEN}"
+RUN if [ -n "$SONAR_TOKEN" ]; then \
+    export PATH="$PATH:/root/.dotnet/tools"; \
+    dotnet sonarscanner end /d:sonar.login="${SONAR_TOKEN}"; \
+    fi
```

Comment:
Only run the scanner if SONAR_TOKEN is set - helps with local testing / debugging, runs the same on CI